client/components/user-home.js

```diff
@@ -1,34 +1,62 @@
-import React from 'react'
+import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 import {connect} from 'react-redux'
+import {fetchUserData} from '../store/userReducer'
 
-/**
- * COMPONENT
- */
-export const UserHome = props => {
-  const {email} = props
+export class UserHome extends Component {
+  componentDidMount() {
+    this.props.getUserData(1)
```

Comment:
Cannot get this user data until we finish login... will need another ticket for this.